package.json

```diff
@@ -30,10 +30,7 @@
   },
   "dependencies": {
     "app2png": "^1.0.0",
-    "async": "^2.1.4",
     "fuzzyfind": "^2.0.0",
-    "iconutil": "^1.0.2",
-    "mkdirp": "^0.5.1",
-    "simple-plist": "^0.2.1"
```

Comment:
I noticed the `mdfind` adapter is using `fileName` as the app name. 

```js
  constructor (filePath, fileName) {
    this.path = filePath
    this.name = fileName
    this.iconPath = path.join(
      currentWorkingDirectory,
      `./data/icons/${path.basename(this.path)}.png`
    )
  }

...
  toJson () {
    return {
      id: this.path,
      icon: this.icon(),
      title: this.name,
      subtitle: this.path,
      value: this.path,
    }
  }
```


Should we try to get the `CrAppModeShortcutName`, `CFBundleDisplayName` or `CFBundleName` by `simple-plist` first?

```js
  title () {
    const fileName = this.name.split('.')[0]
    if (this.isAppMac()) {
      return this.info.CrAppModeShortcutName || this.info.CFBundleDisplayName || this.info.CFBundleName || fileName
    } else {
      return fileName
    }
  }
```